app/helpers/route_overrides_helper.rb

```diff
@@ -0,0 +1,40 @@
+module RouteOverridesHelper
+  class << self
+
```

Comment:
If I remove the empty line before `private` and run `rubocop` locally, there's no warning.